aws_lambda_powertools/utilities/data_classes/event_bridge_event.py

```diff
@@ -67,3 +68,22 @@ def detail(self) -> Dict[str, Any]:
     def replay_name(self) -> Optional[str]:
         """Identifies whether the event is being replayed and what is the name of the replay."""
         return self["replay-name"]
+
+    @property
+    def event_size(self) -> int:
+        """Returns the EventBridge Event Size.
+
+        URLs:
+            https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-putevent-size.html
+        """
+
+        size = 0
+        if self["time"] is not None:
```

Comment:
Can't recall if DictWrapper will give you None or raise KeyError if not present -- an unit test here can verify

Reply:
@heitorlessa  - it would produce a key error. However `.get("time")` also works.

Reply:
Funny enough get("time") was how I originally had it.

```
if self.get("time") is not None:
```

Reply:
Thanks @michaelbrewer ! `.get` is the way to go then